src/lakefs_spec/client_helpers.py

```diff
@@ -68,6 +68,13 @@ def get_tags(client: LakeFSClient, repository: str) -> dict:
     return client.tags_api.list_tags(repository=repository)
 
 
+def create_repository(client: LakeFSClient, repository_name: str, storage_namespace: str) -> None:
+    repository_creation = RepositoryCreation(
+        name=repository_name, storage_namespace=storage_namespace
+    )
+    client.repositories_api.create_repository(repository_creation=repository_creation)
```

Comment:
I started to return all API results from creation calls like this in case the user wants to do something with them (logging etc.) - does `create_repository` return an informative object that we could expose?

Reply:
I would return the `lakefs_sdk.models.Repository` object from the `create_repository` call as is, and then let the user decide what to do with it.

Oh and can you change the argument to `name` instead of `repository_name`? This is evident by the name of the caller and the repository creation use.